roles/middleware_monitoring_config/templates/kube_state_metrics_alerts.yml.j2

```
@@ -60,6 +60,112 @@ spec:
         expr: |
           (kube_pod_container_status_waiting_reason{reason="ContainerCreating"} * on (namespace, namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key=~".*"}) > 0
         for: 15m
+      - alert: ThreeScaleApicastStagingPod
+        annotations:
+          message: >-
+            3Scale apicast-staging has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"apicast-staging.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleApicastProductionPod
+        annotations:
+          message: >-
+            3Scale apicast-production has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"apicast-production.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleBackendWorkerPod
+        annotations:
+          message: >-
+            3Scale backend-worker has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"backend-worker.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleBackendListenerPod
+        annotations:
+          message: >-
+            3Scale backend-listener has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"backend-listener.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleBackendRedisPod
+        annotations:
+          message: >-
+            3Scale backend-redis has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"backend-redis.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleSystemRedisPod
+        annotations:
+          message: >-
+            3Scale system-redis has no pods in a ready state.
+        expr: |
+          absent(kube_pod_status_ready{namespace="3scale", condition="true", pod=~"system-redis.*"})
+        for: 5m
+        labels:
+          severity: critical
+      - alert: ThreeScaleSystemDatabasePod
```

There's a few db pods - should we call out this is the alert for the mysql one?

```suggestion
      - alert: ThreeScaleSystemMySQLPod
```

---

Done